.github/workflows/ci.yml

```diff
@@ -9,6 +9,14 @@ on:
 jobs:
   ci:
     runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:latest
+        ports:
+          - 5432:5432
+        env:
```

Comment:
This is purposefully left blank as this service is used while executing integration tests with the database. Each test accesses the db by instantiating the `TestContext` struct, that automatically connects to the default, always available `postgres` database and creates a test-specific database for each run. Then, thanks to a custom `drop` implementation on the strcut, the database is deleted after the context goes out of scope (i.e. when the test is done). That's why we don't need a specific db name. It would just create a database that we wouldn't use ion this setting.